<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacher</class>
 <widget class="QWidget" name="teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>120</width>
     <height>550</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="gridSize">
    <size>
     <width>0</width>
     <height>91</height>
    </size>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>个人信息</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>课表信息</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>学生信息</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>考试信息</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>教室信息</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>成绩信息</string>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>1050</width>
     <height>550</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="pg_personal">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>561</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>性别</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledit_sex"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>生日</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledit_bitrh"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>学院</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledit_clg"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>教师编号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledit_tno"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>姓名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledit_tname"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>职称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledit_position"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>联系方式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledit_tel"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="pg_lesson">
    <widget class="QTableWidget" name="lesson_table">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>1000</width>
       <height>550</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>158</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>104</number>
     </attribute>
     <row>
      <property name="text">
       <string>1-2节</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3-4节</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5-6节</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7-8节</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9-11节</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>周一</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>周二</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>周三</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>周四</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>周五</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>周日</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="pg_student">
    <widget class="QTableWidget" name="student_table">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>950</width>
       <height>550</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>158</number>
     </attribute>
     <column>
      <property name="text">
       <string>学号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>性别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>籍贯</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>生日</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>班级</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>941</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1,1,3,1,1,3,1,1">
      <item>
       <widget class="QLabel" name="label3">
        <property name="text">
         <string>学院</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="college"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>专业</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="major"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>课程</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="course"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="search_student">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="pg_exam">
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>450</width>
       <height>320</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>64</number>
     </attribute>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>学号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>性别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>班级</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>考试科目</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>考试时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>参加考试情况</string>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>姓名查询</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>学号查询</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>所有学生</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineedit_info">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="search_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>班级</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>科目</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>科目</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>班级</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>所有学生</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="pg_classroom">
    <widget class="QTableWidget" name="classroom_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>450</width>
       <height>350</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>149</number>
     </attribute>
     <column>
      <property name="text">
       <string>教室号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>容量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>使用情况</string>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>99</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>按日期</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按课时</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>99</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="search_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>3</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pg_score">
    <widget class="QTableWidget" name="score_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>450</width>
       <height>320</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>89</number>
     </attribute>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>学号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>性别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>班级</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>成绩</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>441</width>
       <height>54</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4,1,4,1,1">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="label1_2">
          <property name="text">
           <string>班级</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="major_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label2_2">
          <property name="text">
           <string>课程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="course_2"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="search_3">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>成绩升序</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>成绩降序</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>平均分</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>分数段分布</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mongolian Baiti</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>教务管理系统—教师端</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
